packages/bpk-component-boilerplate/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow */
+
+import BpkBoilerplate from './src/BpkBoilerplate';
+
+export default BpkBoilerplate;
```

Comment:
We need to export the props definition (flow) here too.

Reply:
Are you sure? I thought we only exported Flow types that are likely to be used by consumers, e.g. if there was a `ButtonType` type.

Reply:
I think so, props are public so types should be visible. If consumers want to extend our components and we don't export the types they would have to redefine all props.